src/data/restore.ts

```diff
@@ -168,14 +168,18 @@ const restoreElement = (
         fontSize = parseInt(fontPx, 10);
         fontFamily = getFontFamilyByName(_fontFamily);
       }
+      const text = element.text ?? "";
+      const lineCount = text.split("\n").length;
```

Comment:
let's reuse `getTextHeight()` here.

Reply:
my bad, misread the function. I was just seeing that the text split is done here without normalizing the newlines. Let's move out the legacy lineHeight calculation into a `getLegacyLineHeight` so it's clear what we're doing here.

Reply:
(also no reason to be splitting every text regardless of migration, hence factoring into a function)